buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -22,18 +22,18 @@
  * SOFTWARE.
  */
 object Versions {
-    const val kotlin = "1.3.72"
-    const val androidGradle = "3.6.4"
+    const val kotlin = "1.5.21"
+    const val androidGradle = "7.0.1"
```

Comment:
you should bump this to 7.0.2 to match the gradle version

Reply:
AGP is only matching Gradle's first two version numbers, so there is no 7.0.2 and 7.0.1 is just a minor bug hotfix, unrelated to Gradle's 7.0.1/2 https://developer.android.com/studio/releases/gradle-plugin#7-0-0

Reply:
Unfortunately, their documentation is out of date
https://mvnrepository.com/artifact/com.android.tools.build/gradle/7.0.2

Reply:
Oh I see, there was an update for AGP that hasn't been posted on the releases doc yet, updated!